symphony-bdk-extensions/symphony-group-extension/src/test/java/com/symphony/bdk/ext/group/TestApiClient.java

```diff
@@ -0,0 +1,35 @@
+package com.symphony.bdk.ext.group;
+
+import com.symphony.bdk.http.api.ApiClient;
+import com.symphony.bdk.http.api.Pair;
+import com.symphony.bdk.http.api.auth.Authentication;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public abstract class TestApiClient implements ApiClient {
+
+  private Map<String, Authentication> authentications = new HashMap<>();
+
+  @Override
+  public List<Pair> parameterToPairs(String collectionFormat, String name, Object value) {
+    return Collections.singletonList(new Pair(name, value.toString()));
+  }
+
+  @Override
+  public String escapeString(String str) {
+    return str;
+  }
+
+  @Override
+  public Map<String, Authentication> getAuthentications() {
+    return authentications;
```

Comment:
Can't we directly return new HashMap() and get ride of the field authentications?

Reply:
Yes we need them to check that "bearerAuth" is added